sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClient.java

```diff
@@ -488,4 +490,19 @@ private static Proxy proxyOptionsToJavaNetProxy(ProxyOptions options) {
                 return new Proxy(Type.HTTP, options.getAddress());
         }
     }
+
+    private void openUrl(String url) throws IOException {
+        Runtime rt = Runtime.getRuntime();
+
+        String os = System.getProperty("os.name").toLowerCase();
```

Comment:
of non-localized String.toUpperCase() or String.toLowerCase() in com.azure.identity.implementation.IdentityClient.openUrl(String) [com.azure.identity.implementation.IdentityClient] At IdentityClient.java:[line 497] DM_CONVERT_CASE

Reply:
Changed to `toLowerCase(Locale.ROOT)`